src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,107 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double GROWTH_FACTOR = 1.5;
+    private Object[] elements = new Object[DEFAULT_CAPACITY];
```

Comment:
Please use constructor to instantiate this (https://mate-academy.github.io/jv-program-common-mistakes/java-core/generics/storage.html#any-magic-numbers-should-be-constants)